enterprise/server/test/integration/podman/BUILD

```diff
@@ -2,30 +2,26 @@ load("@io_bazel_rules_go//go:def.bzl", "go_test")
 
 package(default_visibility = ["//enterprise:__subpackages__"])
 
-# To run this test with remote execution, add a test_tag_filter:
-# bazel test --config=remote --test_tag_filters=+bare \
-# //enterprise/server/test/integration/podman:podman_test
 go_test(
     name = "podman_test",
     size = "large",
     srcs = ["podman_test.go"],
     exec_properties = {
-        "test.Pool": "bare",
-        "test.use-self-hosted-executors": "true",
-        "test.container-image": "none",
+        "test.workload-isolation-type": "firecracker",
+        "test.EstimatedComputeUnits": "4",
+        "test.EstimatedFreeDiskBytes": "10GB",
+        # Use the executor image since it has podman installed.
+        "test.container-image": "docker://gcr.io/flame-public/buildbuddy-executor-enterprise:enterprise-v2.47.0",
```

Comment:
If a test is passing at a specific git commit, it could potentially start failing at some point in the future if the latest tag is updated. Then things like `git bisect` become more painful. So I think generally it's good to keep our dependencies pinned.

We don't need this to be super up-to-date, just need it to have podman. If we want to test new podman versions then the downside is that we'll have to update this manually, but I think that's an OK tradeoff.

Reply:
Wouldn't we want to know if a new podman version in a new executor image breaks this test, though? Worst case scenario it's a test-only failure, but if not it could conceivably catch real problems, though I admit it's unlikely.

Reply:
Yeah, I think ideally we'd build podman ourselves and depend on it from runfiles rather than depending on a specific docker image. We'll want to figure that out at some point soon, to avoid depending on the third-party podman apt repo.